cinema/urls.py

```diff
@@ -1 +1,17 @@
-# write urls here
+from django.urls import path, include
+from rest_framework import routers
+from rest_framework.urls import app_name
```

Comment:
The import `app_name` from `rest_framework.urls` is not used in this file. Consider removing it to clean up the code.